From a4f65bf89d955f585f141fd3402e51eec53ba562 Mon Sep 17 00:00:00 2001
From: Sebastian Meyer <meyer@b1-systems.de>
Date: Thu, 23 Aug 2018 16:45:27 +0200
Subject: [PATCH 1/1] Fix error with --uninstall option on SUSE

The `-l` option for some `zypper` subcommands is short for
`--auto-agree-with-licenses` and not available on the `zypper rm` command.

(cherry picked from commit fd43730e2bfff17a365ffcf768b6c8db3089b524)
[BSC#1131342]
Signed-off-by: Larry Dewey <ldewey@suse.com>
---
 customize/customize_run.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/customize/customize_run.ml b/customize/customize_run.ml
index f92e9a199..3eacdaca0 100644
--- a/customize/customize_run.ml
+++ b/customize/customize_run.ml
@@ -181,7 +181,7 @@ exec >>%s 2>&1
     | "urpmi" ->  sprintf "urpme %s" quoted_args
     | "xbps" ->   sprintf "xbps-remove -Sy %s" quoted_args
     | "yum" ->    sprintf "yum -y remove %s" quoted_args
-    | "zypper" -> sprintf "zypper -n rm -l %s" quoted_args
+    | "zypper" -> sprintf "zypper -n rm %s" quoted_args
 
     | "unknown" ->
       error_unknown_package_manager (s_"--uninstall")
-- 
2.21.0

